libweb3jsonrpc/Test.cpp

```diff
@@ -135,16 +136,26 @@ bool Test::test_addBlock(std::string const& _rlp)
     return true;
 }
 
-bool Test::test_rewindToBlock(int _number)
+std::string Test::test_importRawBlock(string const& _blockRLP)
 {
     try
     {
-        m_eth.rewind(_number);
-        asClientTest(m_eth).completeSync();
+        ClientTest& client = asClientTest(m_eth);
+        bytes blockBytes = jsToBytes(_blockRLP, OnFailed::Throw);
+        h256 blockHash = BlockHeader::headerHashFromBlock(blockBytes);
+        client.queueBlock(blockBytes, true);
```

Comment:
Why do you do this now instead of `TestClient::addBlock` ? `addBlock` also notified `EthreumHost` to propagate new block over network, not it doesn't do that

Reply:
oh. now I think I remember where it was used. in experemental rpc tests? 
the issue is that addBlock was not working properly. we could notify ethereum host from importRawBlock as well

Reply:
I don't know where it was used.

Maybe better move this logic into `ClientTest` and leave only call to it in `Test::test_importRawBlock`

Reply:
Maybe just move new stuff to `TestClient::addBlock` and call from here